doc/blocks.xml

```diff
@@ -173,3 +173,27 @@ gap> DegreeOfBlocks(blocks);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="CanonicalBlocks">
+  <ManSection>
+    <Attr Name = "CanonicalBlocks" Arg = "blocks"/>
+    <Returns>Blocks of a bipartition.</Returns>
+    <Description>
+      For the blocks of a bipartition <A>blocks</A> the function
+      <C>CanonicalBlocks</C> returns a canonical represnetative of the
+      <A>blocks</A>.
+      <P/>
+
+      In particular, let <C>C</C> be a class of blocks such that for every pair
```

Comment:
Ah. I was reading this to mean that `C` was on of the parts of the blocks <A>blocks</A>.